core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -345,7 +358,8 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
           sparkProperties,
           SparkEnv.get.securityManager.getIOEncryptionKey(),
           Option(delegationTokens.get()),
-          rp)
+          rp,
+          CoarseGrainedSchedulerBackend.this.logLevel)
```

Comment:
Nit: I do not think we need "CoarseGrainedSchedulerBackend.this." here, do we?

Reply:
Not needed. Fixed